stories/action-sheet/action-sheet.stories.js

```diff
@@ -0,0 +1,200 @@
+            aria-label="More Options" aria-controls="actionSheetPhone" aria-expanded="false" 
+            aria-haspopup="true" onclick="toggleClass('actionSheetPhone', 'fd-action-sheet__wrapper--active');">
+        <i class="sap-icon--settings"></i>
+    </button>
+    <div class="fd-action-sheet__wrapper" id="actionSheetPhone">
+        <h6 class="fd-action-sheet__title">Press cancel to hide action sheet</h6>
+        <ul class="fd-action-sheet fd-action-sheet--mobile" role="list" aria-label="List of contextual options">
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--attachment"></i>
+                    <span class="fd-button__text">Upload file</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--add"></i>
+                    <span class="fd-button__text">Assign user</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--print"></i>
+                    <span class="fd-button__text">Print</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--download"></i>
+                    <span class="fd-button__text">Download</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left"
+                        onclick="toggleClass('actionSheetPhone', 'fd-action-sheet__wrapper--active');">
+                    <span class="fd-button__text">Decide Later</span>              
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--negative"
+                        onclick="toggleClass('actionSheetPhone', 'fd-action-sheet__wrapper--active');">
+                    <span class="fd-button__text">Cancel</span>
+                </button>
+            </li>
+        </ul>
+    </div>
+</div>
+`;
+
+actionSheetPhone.parameters = {
+    docs: {
+        iframeHeight: 800,
+        storyDescription: 'On smartphones the action sheet is wrapped additionally by `fd-action-sheet__wrapper`.'
```

Comment:
To display the action sheet on mobile screens, it is additionally wrapped by `fd-action-sheet__wrapper` together with the `id="actionSheetPhone"`.

Let me know if it makes sense.

The only thing I would suggest about the code is to change "phone" to "mobile" because that's what we've called it so far in the docs.

Reply:
If the ID doesn't really help our readers and is only for us internally, this needs to be rewritten. If we could go over a call to discuss what to include here, feel free to ping me on Slack! Or if it's a short explanation, you can just answer the comment above :)

Reply:
"To display the action sheet on mobile screens, the container requires a `fd-action-sheet--mobile` modifier class  and is additionally wrapped by `fd-action-sheet__wrapper`."